### PR TITLE
Add Beijng file

### LINKED ISSUE #4: learnereveryday tracks progress on GH Flow
- [x] create a **feature branch**
- [x] add some **commits**
- [x] collaborate and add additional commits
- [ ] collaborate and add additional commits
- [ ] merge my pull request with 1 :+1:

### PR DESCRIPTION
**Pull request by learnereveryday**
This PR adds the Beijing file to our project. This PR closes #4 

Some ?s:
1. If you've been to Beijing before, what did you like? What did you not like?
2. If you have children, what might be of interest to you? What do you want to know before you visit Beijing?
3. @hollenberry do you know if there's any attractoins unique to Beijing that you don't have in CA?

### CODE REVIEW

File: learnereveryday-beijing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Beijing, China
+
+### Restaurants
+
+- Peking's duck :bird:
+
+### Local Attractions
+
+- Greatwall
```
Comment (githubteacher) [inline learnereveryday-beijing.md:9]: @learnereveryday how long should I plan to spend at the Great Wall?

File: learnereveryday-beijing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Beijing, China
+
+### Restaurants
+
+- Peking's duck :bird:
```
Comment (alanzhong) [inline learnereveryday-beijing.md:5]: :O

**Comment by learnereveryday** [comment]
pls take a look at the new place to visit in Beijing

File: learnereveryday-beijing.md (hunk 2 above)
Comment (learnereveryday) [inline learnereveryday-beijing.md:5]: nice

**Pull request merged**